File: VB/CustomHeaderMenu.vb
Imports Microsoft.VisualBasic
Imports DevExpress.Xpf.Core
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace WpfApplication3
	Public Class CustomHeaderMenu
		Inherits HeaderMenu
		Public Sub Close()
			PopupClose()
		End Sub
	End Class
End Namespace

File: CS/CustomHeaderMenu.cs
using DevExpress.Xpf.Core;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace WpfApplication3 {
    public class CustomHeaderMenu: HeaderMenu {
        public void Close() {
            PopupClose();
        }
    }
}
